### PR TITLE
Add QAbstractItemModel implementation, add servusBrowser GUI tool

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by tribal-tec** [comment]
FYI @eile

**Comment by tribal-tec** [comment]
Updated. Thx @rdumusc for (offline) comments.

File: CMakeLists.txt [hunk 1]
```diff
@@ -39,10 +39,18 @@ else()
   common_package(DNSSD)
 endif()
 common_package(Boost 1.51 COMPONENTS unit_test_framework)
+common_package(Qt5Core)
+common_package(Qt5Widgets)
 common_package(Threads REQUIRED)
 common_package_post()
 
+if(Qt5Core_FOUND)
+  set(SERVUS_DEPENDENT_LIBRARIES Qt5Core)
```
Comment (rdumusc) [inline CMakeLists.txt]: I don't remember how this DEPENDENT_LIBRARIES work, but I suppose it is per project and not per library then.

**Comment by rdumusc** [comment]
+1, very good now

File: CMakeLists.txt (hunk 1 above)
Comment (tribal-tec) [inline CMakeLists.txt]: Yes, per project. Into influences the ServusConfig.cmake file in this case.

**Pull request merged**